inc/security.php

```diff
@@ -56,12 +56,12 @@ function set_content_security_policy() {
 
 	$csp_allowed = array(
 		"default-src 'self'",
-		"script-src 'self' 'unsafe-inline' 'unsafe-eval' https://piwik.wikimedia.org https://stats.wp.com https://pixel.wp.com https://www.youtube.com https://player.vimeo.com http://localhost https://localhost http://localhost:8080",
-		"frame-src 'self' https://www.youtube.com https://player.vimeo.com",
-		"style-src 'self' 'unsafe-inline'",
-		"img-src 'self' data: https://piwik.wikimedia.org https://wikipedia.org https://upload.wikimedia.org",
-		"font-src 'self' data:",
-		"connect-src 'self' wss://public-api.wordpress.com https://*.wikipedia.org",
+		"script-src 'self' 'unsafe-inline' 'unsafe-eval' https://piwik.wikimedia.org https://stats.wp.com https://pixel.wp.com https://www.youtube.com https://player.vimeo.com http://localhost https://localhost http://localhost:8080 https://s0.wp.com",
```

Comment:
Since we can use wildcards for subdomains, would it make this simpler to just allow `https://*.wikimedia.org` and `https://*.wp.com`?

Reply:
Done @goldenapples , thanks!